packages/vuelidate/src/index.js

```diff
@@ -150,12 +152,12 @@ export function useVuelidate (validations, state, globalConfig = {}) {
             resultsCache,
             globalConfig,
             instance: instance.proxy,
-            externalResults: instance.proxy.vuelidateExternalResults
+            externalResults: $externalResults || instance.proxy.vuelidateExternalResults
```

Comment:
Thanks for doing it! I really like how responsive you are to the needs of community!

Reply:
Note this will still not work for Vue 2, without pre-defining the keys. This is a Vue 2 limitation of the reactivity API.

Reply:
I understand, but I can start with `ref(null)`, return `$externalResults`  from the `setup` hook, and then replace that `null` with any other value and the reference will reflect it, like so `this.$externalResults = serverResponse`.